.eslintrc.js

```diff
@@ -7,12 +7,29 @@ module.exports = {
     {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
+      rules: {
+        // TODO: Move this to our shared config
+        'no-invalid-this': 'off',
+        '@typescript-eslint/no-invalid-this': ['error'],
+      },
     },
 
     {
       files: ['*.js'],
       extends: ['@metamask/eslint-config-nodejs'],
     },
+
+    {
+      files: ['*.test.ts', '*.test.js', './tests/setupAfterEnv.ts'],
+      extends: ['@metamask/eslint-config-jest'],
+    },
+
+    {
+      files: './tests/setupAfterEnv.ts',
+      parserOptions: {
+        sourceType: 'script',
```

Comment:
Hmm. Not sure I follow why this was needed :thinking: That module even has an export.

Reply:
Ah, good catch. I think I added the `export {}` after I added this based on a different warning I was getting. I'll remove this.

Reply:
Fixed in [5229e84](https://github.com/MetaMask/eth-block-tracker/pull/103/commits/5229e84e769c6856ab51fe9421ab6b9b41005f82).